build/packer/ansible/roles/podman-setup/tasks/main.yml

```
@@ -0,0 +1,56 @@
+- name: pull the PMM image
+  command: podman pull {{ pmm_server_image_name }}
+
+- name: get srv directory path
+  shell: podman image inspect {{ pmm_server_image_name }} | jq -r .[0].GraphDriver.Data.UpperDir
+  register: image_path
+
+- name:
+  copy:
+    src: "{{ image_path.stdout }}/srv/"
+    dest: /srv/
+    mode: preserve
+    remote_src: yes
+
+- name: Copy content of 'srv' directory from image
+  synchronize:
+    src: "{{ image_path.stdout }}/srv/"
+    dest:  /srv/
+    recursive: yes
+  delegate_to: "{{ inventory_hostname }}"
+
+- name: Set AMI distribution
+  copy:
+    content: |
+      ami
+    dest: /srv/pmm-distribution
+
+- name: Create user-specific systemd directory
+  file:
+    path: /home/admin/.config/systemd/user/
+    state: directory
+    owner: admin
+    group: admin
+    mode: '0755'
+
+- name: Copy systemd service file to user-specific directory
+  template:
+    src: pmm-server.service
+    dest: /home/admin/.config/systemd/user/
+    owner: admin
+    group: admin
+    mode: '0644'
+
+- name: Get user ID of admin user
+  command: sudo id -u admin
+  register: admin_user_id
+
+- name: Enable linger for the admin user
+  command: sudo loginctl enable-linger {{ admin_user_id.stdout }} 
```

```suggestion
  command: loginctl enable-linger {{ admin_user_id.stdout }} 
  become: true
  become_user: admin
```
Let's use `become` to observe best practices.